tools/harness_capability_profiles.py

```
@@ -466,6 +493,205 @@ def string_list(value: Any) -> bool:
+                path,
+            )
+        ]
+    results: list[CheckResult] = []
+    for index, record in enumerate(records):
+        prefix = f"profile:{harness_id}:harness_extension:{index}"
+        if not isinstance(record, dict):
+            results.append(CheckResult(prefix, False, "harness_extension entry must be a table", path))
+            continue
+        results.extend(validate_string_fields(record, ["id", "name", "scope", "description"], prefix, path))
+        if not string_list(record.get("schema_pressure_ids")):
+            results.append(CheckResult(f"{prefix}:schema_pressure_ids", False, "schema_pressure_ids must be a list of strings", path))
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)